pokemongo_bot/walkers/polyline_walker.py

```diff
@@ -34,7 +34,14 @@ def step(self):
         
         cLat, cLng = self.polyline_walker.get_pos()
         cAlt = self.polyline_walker.get_alt()
-        self.api.set_position(cLat, cLng, cAlt)
+    
+        step_walker = StepWalker(
+            self.bot,
+            cLat,
+            cLng,
+            cAlt
+        )
+        
```

Comment:
This i not going to change the position..

since you replaced .set_position() with just an object initiation but there is nothing calling .step() on the new object